CourseManager/Model/CourseInformation.swift

```diff
@@ -7,26 +7,32 @@
 
 import FirebaseFirestoreSwift
 import Foundation
+import SwiftUI
 
 struct CourseInformation: Identifiable, Codable {
     @DocumentID var id: String? = UUID().uuidString
     var name: String
     var url: String
-    // var status: Status
-    var dateOfCompletion: Date?
+    var gitHubURL: String
+    var status: Status
+    var dateOfCompletion: Date
     var comments: String
 
     enum CodingKeys: String, CodingKey {
```

Comment:
@sghourab why do you use coding keys?

Reply:
Deleted, Not sure why I left them in. I know they're usually for when keys in database/API are different to those needed in app.